integration_tests/src/bin/benchmark/args.rs

```diff
@@ -99,6 +99,104 @@ impl From<EncoderPreset> for ffmpeg_h264::EncoderPreset {
     }
 }
 
+#[derive(Debug, Clone, Copy, clap::ValueEnum)]
+#[clap(rename_all = "snake_case")]
+pub enum ResolutionPreset {
+    Uhd,
+    Qhd,
+    Fhd,
+    Hd,
+    Sd,
+}
+
+impl std::str::FromStr for ResolutionPreset {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "uhd" => Ok(ResolutionPreset::Uhd),
+            "qhd" => Ok(ResolutionPreset::Qhd),
+            "fhd" => Ok(ResolutionPreset::Fhd),
+            "hd" => Ok(ResolutionPreset::Hd),
+            "sd" => Ok(ResolutionPreset::Sd),
+            _ => Err(
+                "invalid resolution preset, available options: sd, hd, fhd, qhd, uhd".to_string(),
+            ),
+        }
+    }
+}
+
+#[derive(Debug, Clone, Copy)]
+pub enum ResolutionConstant {
+    Preset(ResolutionPreset),
+    Value(u32, u32),
```

Comment:
```suggestion
    Value(Resolution),
```
That's how I'd do it, but it's up to you